### PR TITLE
U17: Build avatar into game using API

### PR DESCRIPTION
**Pull request by paul891112**
PlayerManager.getAll() invokes red line in VS Code: "Object is possibly null"

Tested with three players ( each on a different browser ). Every players sees the same avatars at the same position. Basic gameplay should still work.

### CODE REVIEW

File: src/scenes/gameuiscreen.ts [hunk 1]
```diff
@@ -68,9 +71,23 @@ export class GameUiScreen extends Phaser.Scene {
...
+                `https://api.dicebear.com/8.x/pixel-art/svg?seed=${player.name}&size=${GameUiScreen.profilePixels}`,
+            );
+        }
     }
 
+    // In displayProfiles, use: this.add.image(x,y,PlayerManager.getAvatarVariableByPlayer(allPlayers[i]));
+
     public create(): void {
+        //demo
+        this.add.image(100, 100, "avatar0");
```
Comment (C0DECYCLE) [inline src/scenes/gameuiscreen.ts:89]: We will merge this here first and then adapt @Croxsy commit.

File: src/scenes/gameuiscreen.ts [hunk 2]
```diff
@@ -68,9 +71,23 @@ export class GameUiScreen extends Phaser.Scene {
             this.load.image(`tile${i}`, `assets/tiles/tile${i}.png`);
         }
         this.load.image("trashCan", "assets/buttons/trashcan.png");
+        // Can't get rid of error message "Object is possibly null"
```
Comment (C0DECYCLE) [inline src/scenes/gameuiscreen.ts:74]: Why not? getAll() returns a nullable... same as always. Just make a const, assert and then use. Look in the code for examples.

File: src/scenes/gameuiscreen.ts [hunk 3]
```diff
@@ -68,9 +71,23 @@ export class GameUiScreen extends Phaser.Scene {
...
+        for (let i: int = 0; i < PlayerManager.getAll().length; i++) {
+            let player: Player = PlayerManager.getAll()[i];
```
Comment (C0DECYCLE) [inline src/scenes/gameuiscreen.ts:76]: by those 2 usages, use the const you will make above.

**Review by C0DECYCLE — CHANGES_REQUESTED** [review]
Cool thank you really much @paul891112 for your work dealing with the API trouble! Of course as well here with the super sleek implementation.

File: src/scenes/gameuiscreen.ts (hunk 3 above)
Comment (paul891112) [reply src/scenes/gameuiscreen.ts:76]: Not sure what you mean, let me try

File: src/scenes/gameuiscreen.ts (hunk 2 above)
Comment (paul891112) [reply src/scenes/gameuiscreen.ts:74]: Good then i understood you correctly

**Review by C0DECYCLE — APPROVED** [review]

**Review by Patertuck — APPROVED** [review]

**Pull request merged**